github.go

```diff
@@ -54,37 +56,57 @@ func runGitHubHandler(c *gin.Context) {
 
 	importer.Token, err = importer.getToken(i)
 	if err != nil {
-		c.Error(err)
-		c.JSON(http.StatusBadRequest, fmt.Sprintf("Unable to get token. %+v", err))
+		ginutil.WriteProblemError(c, err, problem.Response{
+			Type: "prob/provider/github/getting-token-error",
+			Title: "Error getting token.",
+			Status: http.StatusBadRequest,
+			Detail: "Unable to get token.",
+		})
 		return
 	}
 
 	importer.Provider, err = importer.getProvider(i, importer.Token)
 	if err != nil {
-		c.Error(err)
-		c.JSON(http.StatusBadRequest, fmt.Sprintf("Unable to get provider. %+v", err))
+		ginutil.WriteProblemError(c, err, problem.Response{
+			Type: "prob/provider/github/getting-provider-error",
+			Title: "Error getting GitHub provider",
+			Status: http.StatusBadRequest,
+			Detail: "Unable to get provider.",
+		})
 		return
 	}
 
 	importer.GithubClient, err = importer.initGithubConnection()
 	if err != nil {
-		c.Error(err)
-		c.JSON(http.StatusBadRequest, fmt.Sprintf("Unable to init github connection. %+v", err))
+		ginutil.WriteProblemError(c, err, problem.Response{
+			Type: "prob/provider/github/connection-error",
+			Title: "Error connecting to GitHub.",
+			Status: http.StatusBadRequest,
+			Detail: "Unable to initiate connection to GitHub.",
+		})
 		return
 	}
 
 	if i.ProjectID != 0 || i.Project != "" {
 		err = importer.importProject(i)
 		if err != nil {
-			c.Error(err)
-			c.JSON(http.StatusBadRequest, fmt.Sprintf("Unable to import project. %+v", err))
+			ginutil.WriteProblemError(c, err, problem.Response{
+				Type: "prob/provider/github/import-project-error",
```

Comment:
Missing leading slash in Type

```suggestion
				Type: "/prob/provider/github/import-project-error",
```